### ISSUE TITLE
Improve accessibility of UIKit Components

### ISSUE THREAD

**Issue by jsccosta** [lumada-design/hv-uikit-react#697]
Some of the current components have been identified as having Accessibility issues. The components identified need to be reviewed in order to determine how to best resolve these accessibility faults; 
if a specific type of fault is recurrent it makes sense to define a strategy that encompasses all components having this issue.
The components that have been identified as having issues, are listed below along with the issue number, if it exists.
Besides determining how to best resolve the issues, we should try to define tests that would check for accessibility during the dev process.

#### Components to review

 :black_square_button: - Banner Component - [Issue 678](https://github.com/pentaho/hv-uikit-react/issues/678) &  Breadcrumb component - [Issue 678](https://github.com/pentaho/hv-uikit-react/issues/678): This issue refers to the lack of a name available to an accessibility API, which stems from the Button pattern being used in both components. In these scenarios, both are using a built-by-hand component. First, it is recommended that this ad-hoc button implementation be replaced with the ready made button component in our UIKit, and that we add the necessary that would make the button accessible. The current view of the MUI does not seem to have a prop that we can leverage in order to pass in aria-label(for instance), so it may be necessary to specify a new property in our Button Component that would allow for the definition of labels that would handle this accessibility issue.

 :black_square_button: - Toggle Switch - [Issue 604](https://github.com/pentaho/hv-uikit-react/issues/604): This issue has been looked at and a fix proposed. It is currently on hold as one of the accessibility faults has to do with font contrast and we are waiting for new developments from the design team that should resolve these font contrast issues. It is recommended that the component be double checked in order to infer if there is a better way of ensuring that the component is accessible, without relying on the inputProps available in the MUI library.
**Currently being checked by @jsccosta** 

:ballot_box_with_check:  - Checkbox - [Issue 681](https://github.com/pentaho/hv-uikit-react/issues/681):

:ballot_box_with_check: - ~~Multibutton - [Issue 683](https://github.com/pentaho/hv-uikit-react/issues/683):~~ Required inclusion of html title attribute. - Checked by @jsccosta

:ballot_box_with_check:  - Breadcrumb - [Issue 688](https://github.com/pentaho/hv-uikit-react/issues/688): Described as having problem with id attributes. Need to determine if this is a problem of the component implementation itself, or if it is related to the way the samples are being put together in the story book.
